docs/rapidstart.md

```diff
@@ -0,0 +1,717 @@
+│                 ID                 │  NAME   │         EMAIL          │EXTERNAL_ID│ STATE │PICTURE_URL│ CREATED_AT  │ UPDATED_AT  │
+├────────────────────────────────────┼─────────┼────────────────────────┼───────────┼───────┼───────────┼─────────────┼─────────────┤
+│c6e354c9-ec27-4815-9212-4bdb4fdbfea4│Admin    │admin                   │null       │ENABLED│null       │1727198813231│1727220886536│
+├────────────────────────────────────┼─────────┼────────────────────────┼───────────┼───────┼───────────┼─────────────┼─────────────┤
+```
+
+### Try to log in with no user in the local database
+
+As noted earlier, Unity Catalog has a local database that contains the users allowed to access it. While an administrative account was created, no user account has been created yet. Thus, the following command **will fail**.
+
+```bash
+bin/uc auth login
+```
+
+Your browser will open and you will authenticate with the specific Google identity (e.g., bobbie@rocinante).  While authenticated, the user account has not been added to the local database.  Hence the command will fail with the output similar to the following:
+
+```bash
+Listening on port: 61918
+Attempting to open the authorization page in your default browser.
+If the browser does not open, you can manually open the following URL:
+
+https://accounts.google.com/o/oauth2/auth?client_id=$$.apps.googleusercontent.com&redirect_uri=http%3A%2F%2Flocalhost%3A61918&response_type=code&scope=openid+profile+email&state=$$
+
+Received authentication response.
+Received token response.
+Exception in thread "main" java.lang.RuntimeException: io.unitycatalog.client.ApiException: Error authenticating - {"error_code":"INVALID_ARGUMENT","details":[{"reason":"INVALID_ARGUMENT","metadata":{},"@type":"google.rpc.ErrorInfo"}],"stack_trace":null,"message":"User not allowed: bobbie@rocinante"}
+    at io.unitycatalog.cli.UnityCatalogCli.main(UnityCatalogCli.java:168)
+Caused by: io.unitycatalog.client.ApiException: Error authenticating - {"error_code":"INVALID_ARGUMENT","details":[{"reason":"INVALID_ARGUMENT","metadata":{},"@type":"google.rpc.ErrorInfo"}],"stack_trace":null,"message":"User not allowed: bobbie@rocinante"}
+    at io.unitycatalog.cli.AuthCli.doExchange(AuthCli.java:102)
+    at io.unitycatalog.cli.AuthCli.login(AuthCli.java:62)
+    at io.unitycatalog.cli.AuthCli.handle(AuthCli.java:46)
+    at io.unitycatalog.cli.UnityCatalogCli.main(UnityCatalogCli.java:124)
+```
+
+Note, `bobbie@rocinante` is not a real Google Identity account, please replace this with your own.
+
+### Add user account to the local database
+
+To fix this error, let’s start by adding your user account to the UC local database.  Use the CLI with the admin token to add your account to the local database.
+
+```bash
+bin/uc --auth_token $(cat etc/conf/token.txt) user create --name "Bobbie Draper" --email bobbie@rocinante
+```
+
+### Try to log in with your user account {#try-to-log-in-with-your-user-account}
```

Comment:
The ##try-to-log-in-with-your-user-account is being rendered.

Reply:
Thanks for catching this - fixed!